### DISCUSSION TITLE
Synchronize round interval

### DISCUSSION THREAD

**Discussion by Pewibe** [jomjol/AI-on-the-edge-device#3040] [Q&A] [answered]
Is it possible to synchronize the round interval / sending of a MQTT message with absolute Time? For example a value with round interval 15 min should be send at XX:00, XX:15, ... I don't know how to achieve that.
If not possible please add it as feature request.

**Comment by caco3** [comment, marked as answer]
No, this is currently not possible.

**Comment by Slider0007** [comment]
Workaround : You could trigger the cycle start externally by MQTT or REST API.

**Comment by Pewibe** [comment]
--> Please add it as feature request.
